src/android/libs/SmartStore/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
       renderscript.srcDirs = ['src']
       res.srcDirs = ['res']
       assets.srcDirs = ['assets']
-      jniLibs.srcDir '../../external/sqlcipher/libs'
+      jniLibs.srcDir 'libs'
```

Comment:
I think I fixed this a while ago on ```unstable```. @coderReview Would you mind rebasing this pull request off the latest from ```unstable```. I'll run some tests locally after and then get it merged. Thanks!